cloud/deployment/workerqueue/workerqueue.go

```diff
@@ -245,7 +245,7 @@ func IsHostedCeleryWorkerQueueInputValid(requestedWorkerQueue *astro.WorkerQueue
 		errorMessage = fmt.Sprintf("max worker count must be between %d and %d", defaultOptions.MaxWorkerCount.Floor, defaultOptions.MaxWorkerCount.Ceiling)
 		return fmt.Errorf("%w: %s", errInvalidWorkerQueueOption, errorMessage)
 	}
-	if !(requestedWorkerQueue.WorkerConcurrency >= machineOptions.ConcurrentTasks) ||
+	if !(requestedWorkerQueue.WorkerConcurrency >= 1) ||
```

Comment:
I am guessing this is not intentional and was just done for testing

Reply:
no it was...actually a bug fix for https://github.com/astronomer/astro-cli/issues/1267#issuecomment-1607054459

Reply:
should we make it a constant then or leave some explanation?

Reply:
Added a comment.